monitoring/datadog_event.py

```diff
@@ -30,14 +36,17 @@
 - "Allows to post events to DataDog (www.datadoghq.com) service."
 - "Uses http://docs.datadoghq.com/api/#events API."
 version_added: "1.3"
-author: "Artūras `arturaz` Šlajus (@arturaz)"
+author: "Naoya Nakazawa (@n0ts)"
```

Comment:
Please just make a list of authors e.g. 

~~~yaml
author: 
  - "Artūras `arturaz` Šlajus (@arturaz)"
  - "Naoya Nakazawa (@n0ts)"
~~~